src/objclass/class_api.cc

```diff
@@ -648,14 +636,16 @@ int cls_gen_rand_base64(char *dest, int size) /* size should be the required str
 
   ret = cls_gen_random_bytes(buf, sizeof(buf));
   if (ret < 0) {
-    lgeneric_derr(ch->cct) << "cannot get random bytes: " << ret << dendl;
+    lgeneric_derr(ClassHandler::get_instance().cct)
```

Comment:
probably we could 
```c++
#define dout_context ClassHandler::get_instance().cct
```

and use `dout()` instead?

Reply:
Yeah, good idea. Switched to the shorter debugs.